aci/resource_aci_bfdmhifpol.go

```diff
@@ -0,0 +1,259 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAciBfdMultihopInterfacePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciBfdMultihopInterfacePolicyCreate,
+		UpdateContext: resourceAciBfdMultihopInterfacePolicyUpdate,
+		ReadContext:   resourceAciBfdMultihopInterfacePolicyRead,
+		DeleteContext: resourceAciBfdMultihopInterfacePolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciBfdMultihopInterfacePolicyImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"tenant_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"admin_state": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"disabled",
+					"enabled",
+				}, false),
+			},
+			"detection_multiplier": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"min_receive_interval": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"min_transmit_interval": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"descr": {
```

Comment:
Update all other references.

```suggestion
			"description": {
```